### DISCUSSION TITLE
Can somebody show me how to use this program

### DISCUSSION THREAD

**Discussion by Rabbitdmr** [skills/.github#334] [General]
It’s my first time can someone who has experience with this program show me what steps I must take to get started

**Comment by sinsukehlab** [comment]
You can view the courses here: https://skills.github.com/
It seems that you're new to GitHub, so starting from the "Introduction to GitHub" course would be a good choice.

**Comment by Rabbitdmr** [comment]
If you require payment then no thank you Sent from my iPhoneOn Aug 24, 2023, at 1:42 AM, Shinsuke Higashiyama aka Cinzeng Zia ***@***.***> wrote:﻿
You can view the courses here: https://skills.github.com/
It seems that you're new to GitHub, so starting from the "Introduction to GitHub" course would be a good choice.

**Reply by sinsukehlab** [reply]
Payment? What are you talking about? All courses are free.

**Reply by Rabbitdmr** [reply]
Don’t be rude to me excuse you

**Reply by Rabbitdmr** [reply]
Im not being rude to you but I can be talk to me with respect or don’t bother talking to me

**Reply by sinsukehlab** [reply]
Hey, do you think of me as a spam account?

**Comment by cmwilson21** [comment]
Thanks for opening a discussion! As @sinsukehlab has already mentioned, the [Skills courses](https://skills.github.com/) are a great place to start. Check out "Introduction to GitHub" for your first course.

Thank you, @sinsukehlab, for helping the community! I'm going to go ahead and close this discussion as "answered" now 💛
